cmd/preview/list.go

```diff
@@ -72,16 +86,72 @@ func executeListPreviews(ctx context.Context, opts ListFlags) error {
 		}
 		return fmt.Errorf("failed to get preview environments: %s", err)
 	}
-	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
-	fmt.Fprintf(w, "Name\tScope\tSleeping\tLabels\n")
+	previewListOutput, err := getPreviewOutput(ctx, opts)
```

Comment:
why not moving the err check just after this line and avoid repeating it inside each switch case?